battlepass/src/lib.rs

```diff
@@ -99,22 +99,71 @@ pub mod pallet {
 			org_id: T::Hash,
 			battlepass_id: T::Hash
 		},
+
+		/// Points updated for user
+		PointsUpdated {
+			by_who: T::AccountId,
+			for_who: T::AccountId,
+			battlepass_id: T::Hash,
+			amount: u32
+		},
+
+		/// New Reward created
+		RewardCreated {
+			reward_id: T::Hash,
+			battlepass_id: T::Hash,
+			level: u8
+		},
+
+		/// Reward claimed by user
+		RewardClaimed {
+			reward_id: T::Hash,
+			claimer: T::AccountId,
+			collection_id: u32,
+			nft_id: NftId
+		},
+
+		/// Reward state updated
+		RewardStateUpdated {
+			reward_id: T::Hash,
+			state: RewardState
+		},
+
+		/// Achievement level added for Battlepass
+		LevelAdded {
```

Comment:
these two events are the same, so we can do a `LevelUpdated` no?

Reply:
I think it's better to distinguish those events. There is no level state, it just exists or not. So `LevelUpdated` may not be so clear in this case.

Reply:
+x / -x not enough?

Reply:
It's about a list of levels (mapping) for Battlepass. E.g.:
```
Level 1 -> 100 points
Level 2 -> 200 points
Level 3 -> 300 points
```
If you want to remove Level 3, you can't just `--3`.